mkdocs.yml

```diff
@@ -0,0 +1,18 @@
+site_name: GCP Ingestion
+nav:
+  - Home: index.md
+  - Edge Ingestion: ingestion-edge/index.md
+  - Beam Ingestion:
```

Comment:
The terms "Edge Ingestion" and "Beam Ingestion" aren't used anywhere else. I think of `ingestion-edge` and `ingestion-beam` as the formal names for the codebases, so ideally those would be the section names. Or perhaps "Edge Service" and "Beam Pipelines" if the codebase names are ruinous to the aesthetics of the sitenav.

Reply:
wfm, this is mainly developer focused so this makes sense